templates/shared/_feedback.html

```diff
@@ -0,0 +1,68 @@
+            </span>
+          </li>
+          <li class="p-inline-list__item" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : '2018 site feedback', 'eventAction' : 'Unsure if I found what I was looking for', 'eventLabel' : 'Site feedback: Unsure', 'eventValue' : undefined });">
+            <span class="p-feedback__icon">
+              <svg xmlns="http://www.w3.org/2000/svg" height="32" width="32"><g transform="translate(1 1)" fill-rule="evenodd" fill="none"><circle stroke-linejoin="round" r="14.904" cx="15" cy="15" stroke="currentColor" stroke-linecap="round" stroke-width="1.508"/><circle fill-rule="nonzero" cy="10.636" cx="10.636" r="1.454" fill="currentColor"/><circle fill-rule="nonzero" cy="10.636" cx="19.364" r="1.454" fill="currentColor"/><path fill-rule="nonzero" fill="currentColor" d="M9.182 19c0-.727.43-1.091 1.293-1.091h9.05c.862 0 1.293.364 1.293 1.091s-.431 1.091-1.293 1.091h-9.05c-.862 0-1.293-.364-1.293-1.091z"/></g></svg>
+            </span>
+          </li>
+          <li class="p-inline-list__item" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : '2018 site feedback', 'eventAction' : 'No I did not find what I was looking for', 'eventLabel' : 'Site feedback: No', 'eventValue' : undefined });">
+            <span class="p-feedback__icon">
+              <svg xmlns="http://www.w3.org/2000/svg" height="32" width="32"><g fill-rule="evenodd" fill="none"><circle stroke-linejoin="round" r="14.904" cx="16" cy="16" stroke="currentColor" stroke-linecap="round" stroke-width="1.508"/><path fill-rule="nonzero" fill="currentColor" d="M21.723 22.459a.729.729 0 0 0-.071-.747c-1.257-1.724-3.339-2.85-5.677-2.85-2.339 0-4.421 1.126-5.678 2.85a.728.728 0 1 0 1.181.852c.984-1.35 2.621-2.249 4.497-2.249 1.875 0 3.512.899 4.496 2.249a.728.728 0 0 0 1.252-.105z"/><circle fill-rule="nonzero" cy="11.636" cx="11.636" r="1.454" fill="currentColor"/><circle fill-rule="nonzero" cy="11.636" cx="20.364" r="1.454" fill="currentColor"/></g></svg>
+            </span>
+          </li>
+        </ul>
+      </div>
+      <div class="p-feedback__result u-hide">
+        <p>Thank you for your response.</p>
+        <p><a href="https://docs.google.com/forms/d/e/1FAIpQLSd1FVXB3MQA5UNqQ4eV3a4xat9J0onPNsz-O8PBL_m6car1LA/viewform?c=0&w=1">Tell us more using this quick online survey&nbsp;&rsaquo;</a></p>
+      </div>
+    </div>
+  </div>
+</div>
+
+<button class="p-button p-feedback__button" id="feedback-button">Site feedback</button>
+
+<script>
+  function feedbackModalActions() {
+    var feedbackIcons = document.querySelectorAll('.p-feedback__icon');
```

Comment:
We should target either js- data attrs or `js-` namespaced classes so that we are decoupling the js hooks from the styling.